Remove this fix once https://github.com/redhat-performance/tuned/commit/a8f2a8306e1bac6cfc739e6753d381bf509c995e is productized.

plugin_sysctl: fixed traceback with modifiers '>', '<' and orig==new

It fixed the following problem e.g. the profile:
[sysctl]
kernel.pid_max=>131072

and if kernel.pid_max is already 131072 Tuned shows traceback.

Resolves: rhbz#1739418

diff --git a/plugins/plugin_sysctl.py b/plugins/plugin_sysctl.py
index bcaead29..b298bfa8 100644
--- a/plugins/plugin_sysctl.py
+++ b/plugins/plugin_sysctl.py
@@ -54,12 +54,13 @@ def _instance_apply_static(self, instance):
 				log.error("sysctl option %s will not be set, failed to read the original value."
 						% option)
 			else:
-				instance._sysctl_original[option] = original_value
 				new_value = self._variables.expand(
 						self._cmd.unquote(value))
 				new_value = self._process_assignment_modifiers(
 						new_value, original_value)
-				_write_sysctl(option, new_value)
+				if new_value is not None:
+					instance._sysctl_original[option] = original_value
+					_write_sysctl(option, new_value)
 
 		storage_key = self._storage_key(instance.name)
 		self._storage.set(storage_key, instance._sysctl_original)
